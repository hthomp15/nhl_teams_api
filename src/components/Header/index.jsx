import React from "react";

function Header() { 
  return (
    <nav className="nav">
     <h1>NHL Reference</h1>
    </nav>
    );
}

export default Header;
