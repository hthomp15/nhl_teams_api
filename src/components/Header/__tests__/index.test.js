import React from "react";
import { render, cleanup } from "@testing-library/react";
import '@testing-library/jest-dom/extend-expect';

import Header from "..";

afterEach(cleanup);

describe("Nav Component", () => {
    it("renders", () => {
        render(<Header />);
    });
    it("matches snapshot DOM node structure", () => {
        const { asFragment } = render(<Header />);
        expect(asFragment()).toMatchSnapshot();
    })
});