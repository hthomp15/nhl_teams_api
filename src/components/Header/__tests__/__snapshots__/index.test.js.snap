// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <nav
    class="nav"
  >
    <h1>
      NHL Reference
    </h1>
    <div>
      Back to Teams
    </div>
  </nav>
</DocumentFragment>
`;
